packages/gestalt/src/Masonry/defaultTwoColumnModuleLayout.test.js

```diff
@@ -388,7 +388,7 @@ describe('two column layout test cases', () => {
     const positionCache = new MeasurementStore<{ ... }, Position>();
     const heightsCache = new HeightsStore();
     const items = [
-      { 'name': 'Pin 0', 'height': 200, 'color': '#E230BA', 'columnSpan': 2 },
+      { 'name': 'Pin 0', 'height': 200, 'color': '#E230BA' },
```

Comment:
why was this changed?

Reply:
To test more positions, the same structure as the previous test, I'm testing when tcm is on index 0 and on index 3